requirements.txt

```diff
@@ -1,4 +1,25 @@
+atomicwrites==1.4.0
+attrs==20.2.0
+certifi==2020.6.20
+chardet==3.0.4
+click==7.1.2
+colorama==0.4.3
+coverage==5.2.1
 fastapi==0.60.2
+h11==0.9.0
+idna==2.10
+iniconfig==1.0.1
+more-itertools==8.5.0
+packaging==20.4
+pluggy==0.13.1
+py==1.9.0
+pydantic==1.6.1
+pyparsing==2.4.7
 pytest==6.0.1
-coverage==5.2.1
```

Comment:
We need coverage. You should not be removing already merged items in requirements unless you have a good reason.